core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -173,10 +199,7 @@ func (env *CouchDBCommonStorageTestEnv) GetName() string {
 
 // Cleanup implements corresponding function from interface TestEnv
 func (env *CouchDBCommonStorageTestEnv) Cleanup() {
-	csdbProvider, _ := env.provider.(*CommonStorageDBProvider)
```

Comment:
This should stay here. These lines of code just cause deletion of databases in couchdb created by tests. Only the last line in this function causes stopping of couchdb container